src/js/util/svg.js

```diff
@@ -50,25 +50,25 @@ define(function (require, exports) {
     };
 
     /**
-     * Gets an array of the CSS class names, one for each layer type in the parameter array
+     * Gets an array of the CSS class names, one for each filter type in the parameter array
      *
      * @private
-     * @param {Array.<string>} layerTypes
+     * @param {Array.<string>} filter
      * @return {Array.<string>}
     */
-    var getSVGClassesFromLayerTypes = function (layerTypes) {
-        if (layerTypes.length === 0) {
+    var getSVGClassesFromFilter = function (filter) {
+        if (filter.length === 0) {
             return [];
         }
         // currently only have icons for layers
-        if (layerTypes.length < 2 || layerTypes.join(" ").indexOf("layer") === -1) {
+        if (filter.length < 2 || filter.join(" ").indexOf("layer") === -1) {
```

Comment:
Not happy with this layer dependency here... but we can come back to it when we add more results with icons.